Source/Scene/Scene.js

```diff
@@ -2793,16 +2790,22 @@ define([
      * }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
      *
      * @param {Cartesian2} windowPosition Window coordinates to perform picking on.
+     * @param {Number} [optional] width of the pick rectangle
+     * @param {Number} [optional] height of the pick rectangle
      * @returns {Object} Object containing the picked primitive.
      *
      * @exception {DeveloperError} windowPosition is undefined.
      */
-    Scene.prototype.pick = function(windowPosition) {
+    Scene.prototype.pick = function(windowPosition, width, height) {
         //>>includeStart('debug', pragmas.debug);
         if(!defined(windowPosition)) {
             throw new DeveloperError('windowPosition is undefined.');
         }
         //>>includeEnd('debug');
+        // override the rectangle dimensions if defined
+        var rectangleWidth = width || 3.0;
+        var rectangleHeight = height || width || 3.0;
+        var scratchRectangle = new BoundingRectangle(0.0, 0.0, rectangleWidth, rectangleHeight);
```

Comment:
Similar to how `scratchRectangle.x` and `.y` are set always, you could set `.width` and `.height` right below.